splinterd/src/daemon.rs

```diff
@@ -1318,9 +1318,12 @@ impl SplinterDaemonBuilder {
             CreateError::MissingRequiredField("Missing field: initial_peers".to_string())
         })?;
 
-        let display_name = self.display_name.ok_or_else(|| {
-            CreateError::MissingRequiredField("Missing field: display_name".to_string())
-        })?;
+        let display_name = self
```

Comment:
This should also remain an option and use the node id that is created in SplinterDaemon.start() if display name is not set.